capnp/src/private/layout.rs

```diff
@@ -3102,33 +3102,33 @@ impl <'a> StructBuilder<'a> {
             return Ok(())
         }
 
-        unsafe {
+        
```

Comment:
If we remove this `unsafe` block then the following lines should be less indented.